### PR TITLE
chore(backend): add receiver to quote/outgoing payment

### LINKED ISSUE #354: Update quote/outgoing payment receiver
- https://github.com/interledger/open-payments/pull/128

Remove `receivingAccount` from quote options.
Replace `receivingPayment` with `receiver`

### PR DESCRIPTION
**Pull request by wilsonianb**
## Changes proposed in this pull request

- Do not create quote with receivingAccount.
- Do not manage completing receiver incoming payment.

## Context

- fixes #354 

## Checklist

- [x] Related issues linked using `fixes #number`
- [x] Tests added/updated
- [x] Documentation added
- [x] Make sure that all checks pass

### CODE REVIEW

File: packages/backend/src/graphql/resolvers/liquidity.test.ts [hunk 1]
```diff
@@ -1603,13 +1603,9 @@ describe('Liquidity Resolvers', (): void => {
...
           description: 'description!'
         })
-        const { id: receivingAccountId } = await accountService.create({
-          asset: account.asset
-        })
-        const config = await deps.use('config')
         payment = await createOutgoingPayment(deps, {
           accountId,
-          receivingAccount: `${config.publicHost}/${receivingAccountId}`,
+          receiver: incomingPayment.id,
```
Comment (sabineschaller) [inline packages/backend/src/graphql/resolvers/liquidity.test.ts:1608]: Why is the path not needed anymore? I didn't see a change in `createOutgoingPayment`.

Comment (wilsonianb) [reply packages/backend/src/graphql/resolvers/liquidity.test.ts:1608]: It's not needed when `validDestination: false`, but I'll change it.

File: packages/backend/src/open_payments/payment/outgoing/lifecycle.ts [hunk 2]
```diff
@@ -145,42 +143,6 @@ const handleCompleted = async (
...
-        deps.logger.warn(
-          {
-            error: err.message
-          },
-          'incoming payment PUT request failed'
-        )
-      }
-      throw LifecycleError.ReceivingPaymentError
-    }
-  }
```
[the remark this answers is not in the document]
Comment (wilsonianb) [reply packages/backend/src/open_payments/payment/outgoing/lifecycle.ts:183]: We were previously only completing it if Rafiki created the incoming payment during quote creation with a specified `receivingAccount` and `sendAmount`. (An `incomingAmount` couldn't be determined for such auto-created incoming payments, so it was necessary for the sender complete to it)

Now that Rafiki no longer auto-creates incoming payments, there isn't currently was for it to know to complete a incoming payment after an outgoing payment is sent.

[the remark this answers is not in the document]
Comment (wilsonianb) [reply packages/backend/src/open_payments/payment/outgoing/lifecycle.ts:183]: This actually makes the need to pass access tokens to ilp-pay less pressing
- https://github.com/interledgerjs/interledgerjs/issues/268

**Review by sabineschaller — DISMISSED** [review]

**Review by sabineschaller — APPROVED** [review]

**Pull request merged**